docs/user-guide/configuration-files/strongbox-yaml.md

```diff
@@ -66,6 +66,7 @@ The following is an example of a `strongbox.yaml` configuration file:
               type: hosted
               allowsForceDeletion: true
               checksumHeadersEnabled: true
+              strictChecksum: true
```

Comment:
Please, change this to `strictChecksumValidation`.

Reply:
Sure